zerver/tornado/event_queue.py

```diff
@@ -573,10 +573,14 @@ def request_event_queue(user_profile, user_client, apply_markdown,
                'lifespan_secs': queue_lifespan_secs}
         if event_types is not None:
             req['event_types'] = ujson.dumps(event_types)
-        resp = requests_client.get(settings.TORNADO_SERVER + '/api/v1/events',
-                                   auth=requests.auth.HTTPBasicAuth(
-                                       user_profile.email, user_profile.api_key),
-                                   params=req)
+
+        try:
+            resp = requests_client.get(settings.TORNADO_SERVER + '/api/v1/events',
+                                       auth=requests.auth.HTTPBasicAuth(
+                                           user_profile.email, user_profile.api_key),
+                                       params=req)
+        except requests.adapters.ConnectionError:
+            raise requests.adapters.ConnectionError('Cannot connect to Zulip\'s Tornado service')
```

Comment:
Having a log with `error` priority with additional information, like `Tornado server does not seem to running check port whatever` would be useful here.